assets/js/modules/search-console/util/index.js

```diff
@@ -25,6 +25,7 @@ import { each } from 'lodash';
  * Internal dependencies
  */
 import { changeToPercent, readableLargeNumber } from '../../../util';
+export * from './is-zero-report';
```

Comment:
This feels a _bit_ dangerous to me, because we're letting one file control another's exports. It's not a big deal, but given it's only one function anyway, I'd rather us be explicit.

That said: these big utils files can sometimes make for annoying circular imports or big files. I'm fine with this, but I'm also fine if we import from a specific file. I don't think we're really good about sticking to one way, so I get why you did this 😄

Maybe let's just use an explicit import here instead of `*`?

Reply:
Switched to explicit export of isZeroReport within utils